utilities/CovidDownloader.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+from urllib.request import urlopen
+import json
+import csv
+
+
+class csvDownloader():
+	def __init__(self):
+		pass	
+
+	def find_data(self, countryName):
+		
+
+		df = pd.read_csv(url)
+		global myDict 
+		myDict = df.to_dict()
+
+		global x
+		x = list(myDict['Name'].keys())[list(myDict['Name'].values()).index(countryName)]
```

Comment:
Could you be slightly more descriptive with the variable name? At a glance, I have no idea what x is for